pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -1352,6 +1356,19 @@ func (r *ReconcileClusterDeployment) setClusterStatusURLs(cd *hivev1.ClusterDepl
 	return reconcile.Result{}, nil
 }
 
+// setFakeClusterStatusURLs skips cluster communication and sets fake cluster URLs in status.
```

Comment:
Do you think keeping these fake functions in the same file as actual code is good/ acceptable.

Reply:
I will move to a fake.go.